src/SOS/Strike/strike.cpp

```diff
@@ -3966,7 +3966,6 @@ DECLARE_API(DumpModule)
     DMLOut("Assembly:                %s\n", DMLAssembly(module.Assembly));
 
     ExtOut("BaseAddress:             %p\n", SOS_PTR(module.ilBase));
-    ExtOut("PEAssembly:              %p\n", SOS_PTR(module.PEAssembly));
     ExtOut("ModuleId:                %p\n", SOS_PTR(module.dwModuleID));
     ExtOut("ModuleIndex:             %p\n", SOS_PTR(module.dwModuleIndex));
```

Comment:
```suggestion
```
These can be deleted too. They are also recent additions that did not existed in the golden .NET Framework, they are rarely needed information that can be fetched using regular windbg commands, and they no longer exist as a concept in .NET.